### ISSUE TITLE
Add user guide

### ISSUE THREAD

**Issue by mlutze** [jaschdoc/flix-parsers#35]
Just some examples/walkthrough to making your own parser

**Comment by jaschdoc** [comment]
I agree, but I found a bug that I can't narrow down in https://github.com/jaschdoc/flix-parsers/pull/22. Apparently, there is some combinator (or possibly more) that just succeeds on the first recognized word and then stops parsing the rest. Anyway, my idea was that I'd add some more documentation when it's ready for use :)
